plugins/by-name/rest/default.nix

```
@@ -146,49 +40,20 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
           Refer to the documentation for more information.
         ''
       )
-      (mkRenamedOptionModule
-        (
-          basePluginPath
-          ++ [
-            "result"
-            "formatters"
-          ]
-        )
-        (
-          settingsPath
-          ++ [
-            "result"
-            "behavior"
-            "formatters"
-          ]
-        )
-      )
-      (mkRenamedOptionModule
-        (
-          basePluginPath
-          ++ [
-            "highlight"
-            "enabled"
-          ]
-        )
-        (
-          settingsPath
-          ++ [
-            "highlight"
-            "enable"
-          ]
-        )
+    ]
+    ++ (map
```

```suggestion
    ++ map
```
No need to wrap concatenation segments with `()` usually